examples/platform/esp32/common/Esp32AppServer.cpp

```diff
@@ -114,6 +114,15 @@ void Esp32AppServer::DeInitBLEIfCommissioned(void)
 #endif /* CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING */
 }
 
+void Esp32AppServer::Shutdown()
+{
+    chip::DeviceLayer::Internal::BLEMgr().Shutdown();
```

Comment:
This is done unconditionally? Should it be guarded by BLE related option?

Reply:
Guarded it with `CONFIG_BT_NIMBLE_ENABLED`